planet/scripts/v1.py

```diff
@@ -735,6 +745,18 @@ def cancel_order(order_id, pretty):
 @pretty
 def create_order(pretty, **kwargs):
     '''Create an order'''
+    ids_from_search = kwargs.get('ids_from_search')
+    if ids_from_search is not None:
+        data_search = kwargs.get('ids_from_search').split()
+        if len(data_search) > 0:
+            runner = CliRunner()
+            resp = runner.invoke(quick_search, data_search).output
+            try:
+                id_list=ids_from_search_response(resp)
+            except ValueError:
+                raise click.ClickException('ids_from_search, {}'.format(resp))
+            kwargs['id'] = id_list
```

Comment:
It might be worth putting a note in this client about how `clip` and `id` interact on the backend.
If ids are provided for items that do not intersect the clip AOI, the [tools reference](https://developers.planet.com/docs/orders/tools-reference/#clip) indicates that those items will be discarded. If the `ids_from_search` don't intersect the `clip` you could create an order that would have no results. Does the backend allow such an order to be created? If not, perhaps adding some spatial validation in this client would be worthwhile for now.

Reply:
yeah this is a good callout and kind of one of those UX things that have bounced around orders for some time. the outcome can be 1) all items intersect clip and are delivered, 2) some intersect and only those are delivered, and 3) none intersect and order has no results.

I'm kind of inclined to just update docs here. I sort of feel like the `planet` tool, being a thin front-end wrapper, should always defer to whatever backend for behaviors, or we have to make sure behaviors in the client are always updated to stay in sync with the backend. thoughts?

Reply:
I have updated the docs to clarify the behavior here. marking as resolved (feel free to unresolve, if you think more discussion is needed).